packages/components/src/Chip/Chip.css

```diff
@@ -111,3 +112,76 @@
 .chip.disabled .suffix {
   background-color: var(--color-disabled);
 }
+
+.chipHeading {
+  flex-shrink: 0;
+}
+
+.chipHeading,
+.chipLabel {
+  position: relative;
+  overflow: hidden;
+}
+
+.chipHeading > p,
+.chipLabel > p {
+  overflow: hidden;
+  white-space: nowrap;
+}
```

Comment:
I don't love this but we need to stop the text from wrapping which it will do by default

if I add `maxNumberOfLines` that stops wrapping, but includes ellipsis truncation behavior that I don't wnt

Reply:
Maybe eventually we just... do the `maxNumberOfLInes` truncation with gradients as a rule in Text? I see that we're doing it within DataList as well, and as you've seen with Combobox and you can see in the DataList work, that we're starting to really consistently see these designs calling for gradient-based overflow indicators.

But for now yeah this totally makes sense.